lib/puppet/application/agent.rb

```diff
@@ -164,13 +164,13 @@ def help
 only resources not tagged with the specified tags will be applied.
 Values must be comma-separated.
 
+
 OPTIONS
 -------
 
 Note that any Puppet setting that's valid in the configuration file is also a
 valid long argument. For example, 'server' is a valid setting, so you can
-specify '--server <servername>' as an argument. Boolean settings translate into
-'--setting' and '--no-setting' pairs.
+specify '--server <servername>' as an argument. Boolean settings accept a 'no-' prefix to turn off a behavior, translating into '--setting' and '--no-setting' pairs, such as `daemonize` and `no-daemonize`.
```

Comment:
Could you break the long line so `puppet agent --help` displays within the terminal without scrolling?

The `'no-' prefix` and `'no-daemonize'` wording is a little confusing to be, because it almost sounds like you can pass that on the command line, like `puppet agent no-daemonize`. I think it might be clearer to say `'--no-' prefix` and `--no-daemonize` as that's what a user would need to type on the command line.